src/xserver/mod.rs

```diff
@@ -1,22 +1,229 @@
-use crate::control::{Controller, Eyes};
+use crate::control::{Controller, Eyes, ToBrain, ToController};
 
-pub struct XController;
-pub struct XEyes;
+use std::{ptr, sync::mpsc::{SyncSender, Receiver}, process::Command, time::Instant};
+use x11::xlib::{self, _XDisplay};
+use image::{ImageBuffer, Rgb};
+
+pub struct XController {
```

Comment:
Why not also store the `*mut _XDisplay` that keeps getting passed around to functions?